sdk/metric/manual_reader_test.go

```diff
@@ -69,7 +70,7 @@ func TestManualReaderTemporality(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			rdr := NewManualReader(tt.options...)
-			assert.Equal(t, tt.wantTemporality, rdr.temporality(undefinedInstrument))
+			assert.Equal(t, tt.wantTemporality, rdr.temporality(0))
```

Comment:
```suggestion
			var undefinedInstrument view.InstrumentKind
			assert.Equal(t, tt.wantTemporality, rdr.temporality(undefinedInstrument))
```
nit: i'd prefer using an uninitialized InstrumentKind to an int.

Reply:
+1

Reply:
Done